can-bus/json/GSM/GSM_alerts.json

```
@@ -1,6 +1,3 @@
 {
-    "msg_id": 600,
-    "alerts": [
-        "GSM_ALERT_STARTUP"
-    ]
+    
 }
```

nit: missing blank line
```suggestion
}

```